nomt/src/beatree/allocator/free_list.rs

```diff
@@ -20,6 +20,8 @@ pub struct FreeList {
     portions: Vec<(PageNumber, Vec<PageNumber>)>,
     // Becomes true if something is being popped, false otherwise
     pop: bool,
+    len: usize,
+    fragmented: bool,
```

Comment:
I'd say this really could use some documentation

Reply:
https://github.com/thrumdev/nomt/blob/1d0dcde979786eeb87890a5a7f822e3d0a951b94/nomt/src/beatree/allocator/free_list.rs#L120-L125

like this?

Reply:
I added a comment to direct the reader to this larger section of text.